src/kixi/datastore/filestore/migrators/dynamodb/migrator_20171221_ttl.clj

```diff
@@ -0,0 +1,60 @@
+(ns kixi.datastore.filestore.migrators.dynamodb.migrator-20171221-ttl
+  (:require [kixi.datastore.system :refer [read-config]]
+            [kixi.datastore.application :refer [profile config-location]]
+            [kixi.datastore.filestore.upload-cache.dynamodb :as fsdb]
+            [kixi.datastore.filestore.upload :as fsu]
+            [kixi.datastore.filestore :as fs]
+            [kixi.datastore.dynamodb :as db]
+            [kixi.datastore.cloudwatch :refer [table-dynamo-alarms]]
+            [taoensso.faraday :as far]
+            [taoensso.timbre :as log]
+            [clj-time.core :as t]))
+
+(def ttl-col (db/dynamo-col ::fsu/ttl))
+
+(defn three-days-from-now
+  []
+  (t/in-seconds
+   (t/interval (t/epoch)
+               (t/plus (t/now) (t/hours 72)))))
+
+(defn get-db-config
+  [db]
+  (select-keys db [:endpoint]))
+
+(defn get-alerts-config
+  [profile]
+  (:alerts (read-config @config-location profile)))
+
+(defn up
+  [db]
+  (let [profile (name @profile)
+        conn (get-db-config db)
+        items (not-empty (far/scan conn (fsdb/primary-upload-cache-table profile)))]
```

Comment:
Might want to explore the faraday scan options to make it scan slowly to not push the table over.